docs/FAQ.md

```diff
@@ -14,7 +14,7 @@ Require stack:
     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
     at Function.Module._load (internal/modules/cjs/loader.js:862:27)
     at Module.require (internal/modules/cjs/loader.js:1042:19)
-    at require (internal/modules/cjs/helpers.js:77:18)
+    at require (internal/modules/cjs/removeRecurrentPayments.js:77:18)
```

Comment:
It seems these changes do not apply to this pr :)

Reply:
I have renamed helpers.js to removeRecurrentPayments.js - thats why I renamed this line, just to follow real filenames